### PR TITLE
feat: added WebSocket doc on Cloud/LightDB Section

### PR DESCRIPTION
**Pull request by miguel91it**

### CODE REVIEW

File: docs/cloud/1-services/2-lightdb/3-read-write-data.md [hunk 1]
```diff
@@ -17,6 +17,21 @@ Devices interact with LightDB using the gateways available on the platform. For
 >
 > /.d/temp/active
 
+Another way of interaction with the platform is opening a Real-Time WebSocket connection only to listen to any changes in a device state path:
+
+| Method    | Description                       | Complete Endpoint                                               |
+| --------- | --------------------------------- | --------------------------------------------------------------- |
+| WebSocket | Listening to LightDB Device State | wss://api.golioth.net/v1/ws/projects/{projectId}/devices/{deviceId}/data{/path=\*\*}?x-api-key={API_KEY} |
```
Comment (alvarowolfx) [inline docs/cloud/1-services/2-lightdb/3-read-write-data.md:24]: Users can also use the `jwt` query param and use the JWT based API Keys from our platform.

File: docs/cloud/1-services/2-lightdb/3-read-write-data.md [hunk 2]
```diff
@@ -168,3 +183,45 @@ The final state of the device in this scenario would be like this:
...
+|Parameter | Optional | Description |
+|--------- | -------- | ----------- |
+|`ProjectID` | **true** | ID of the project to be listened |
+|`DeviceID` | **true** | ID of the device to be listened |
+|`Path` to listened to | false | _empty path_ if desired to listen to the root path <br /> _some path_ to listen to specific path |
+|`API_KEY` | **true** | Api Key to authenticate to the Golioth  | -->
+
+For example, for listening to any changes in the `/env` path, connect a WebSocket client to the following endpoint:
+```
+wss://api.golioth.net/v1/ws/projects/local-test/devices/6173155307bbb1c7c9bb158f/data/env?x-api-key=DDb2UQDZXmbFOp3duDm0CThtmQ9RJuPP
```
Comment (alvarowolfx) [inline docs/cloud/1-services/2-lightdb/3-read-write-data.md:202]: be careful putting real/valid API Keys. Also let's change the URL to `api.golioth.io` which is the production URL.

**Review by alvarowolfx — COMMENTED** [review]
I think would be better if this content would go to a new section on the "Reference" tab named "Websocket API", similar to our current Rest API section

https://golioth-docs-dev--pr91-feature-grpc-ws-prox-6zkgt07n.web.app/reference/rest-api/overview

It can have the same structure with: Overview, Authentication and API Endpoints. And for the API Endpoints section, we can make it like the new CoAP/MQTT docs:
https://golioth-docs-dev--pr91-feature-grpc-ws-prox-6zkgt07n.web.app/reference/protocols/coap/lightdb

File: docs/cloud/1-services/2-lightdb/3-read-write-data.md (hunk 1 above)
Comment (miguel91it) [reply docs/cloud/1-services/2-lightdb/3-read-write-data.md]: Could you show me how the JWT is embedded in the connection request? Does it use the same x-api-key parameter or another way?

**Review by alvarowolfx — APPROVED** [review]
LGTM

**Pull request merged**